### PR TITLE
add tests for git repo importing

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by saeedmb8** [comment]
:+1: :+1: :cake: :cake:

File: molo/core/content_migration/helper.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        language_relation.save()
+        print 'created translation', obj.title
+        return instance
+
+    def import_content_for(self, locales):
+        for selected_locale in locales:
+            site_language = get_object_or_404(
+                SiteLanguage, locale=selected_locale.get('site_language'))
+
+            if site_language.is_main_language:
```
Comment (miltontony) [inline molo/core/content_migration/helper.py]: @saeedmb8 so everything in this IF block can move to a new method called `def import_main_language_content(self, site_language):`

File: molo/core/content_migration/helper.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+                main = Main.objects.all().first()
+                for c in self.ws.S(Category).filter(
```
Comment (miltontony) [inline molo/core/content_migration/helper.py]: You could then split this into its own method too called `def import_section_content`

Comment (miltontony) [inline molo/core/content_migration/helper.py]: and do the same for pages below

**Comment by saeedmb8** [comment]
@miltontony ready for review :)

File: molo/core/content_import/helper.py [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+        instance = cls(uuid=obj.uuid, title=obj.title)
+        parent.add_child(instance=instance)
+        language_relation = instance.languages.first()
+        language_relation.language = language
+        language_relation.save()
+        print 'created translation', obj.title
+        return instance
+
+    def import_section_content(self, c, main, site_language):
```
Comment (saeedmb8) [inline molo/core/content_import/helper.py]: @miltontony I added this function

File: molo/core/content_import/helper.py [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+            section = self.get_or_create_translation(
+                SectionPage, c, main, site_language)
+
+        section.description = c.subtitle
+        # TODO: image
+        section.save_revision().publish()
+
+        return section
+
+    def import_page_content(self, p, main, site_language, parent):
```
Comment (saeedmb8) [inline molo/core/content_import/helper.py]: @miltontony and this function

File: molo/core/content_import/helper.py [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+        page.featured_in_latest = is_featured
+        page.featured_in_homepage = is_featured_in_category
+        # TODO: tags (see mk_tags)
+        # TODO: related pages
+        # TODO: image
+        page.save_revision().publish()
+
+        return page
+
+    def import_main_language_content(self, selected_locale, site_language):
```
Comment (saeedmb8) [inline molo/core/content_import/helper.py]: @miltontony and this one

File: molo/core/content_import/helper.py [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+            self.import_section_content(c, main, site_language)
+
+        for p in self.ws.S(Page).filter(
+            language=selected_locale.get('locale')
+        ).order_by('position')[:10000]:
+            # S() only returns 10 results if you don't ask for more
+
+            self.import_page_content(p, main, site_language, 'foo')
+
+    def import_translated_content(self, selected_locale, site_language):
```
Comment (saeedmb8) [inline molo/core/content_import/helper.py]: @miltontony and this one

File: molo/core/content_import/helper.py [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+                        print "Couldn't find", tp.source, (
+                            ArticlePage.objects.all().values('uuid'))
+
+    def import_content_for(self, locales):
+        for selected_locale in locales:
+            site_language = get_object_or_404(
+                SiteLanguage, locale=selected_locale.get('site_language'))
+
+            if site_language.is_main_language:
+                self.import_main_language_content(
```
Comment (saeedmb8) [inline molo/core/content_import/helper.py]: @miltontony added `self.import_main_language_content` in here

File: molo/core/content_import/helper.py [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+                    selected_locale, site_language)
+            else:
+                self.import_translated_content(selected_locale, site_language)
```
Comment (saeedmb8) [inline molo/core/content_import/helper.py]: @miltontony added `self.import_translated_content`

**Pull request merged**